### PR TITLE
feat: Merge bug fixes and changes

### PR DESCRIPTION
**Pull request by gmalmiski**
Thank you for opening a Pull Request!

---

Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open a GitHub issue as a bug/feature request before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: cloud-functions/src/main/java/com/google/fleetevents/lmfs/models/outputs/EtaAssignedOutputEvent.java [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+ * limitations under the License.
+ */
+package com.google.fleetevents.lmfs.models.outputs;
+
+import com.google.fleetevents.common.models.FleetEvent;
+import com.google.fleetevents.common.models.OutputEvent;
+import java.util.Objects;
+
+/** Output event for when an eta is assigned to a task. */
+public class EtaAssignedOutputEvent extends OutputEvent {
```
Comment (ewbao) [inline cloud-functions/src/main/java/com/google/fleetevents/lmfs/models/outputs/EtaAssignedOutputEvent.java:23]: We should probably set the type of the event here?

File: cloud-functions/src/main/java/com/google/fleetevents/odrd/models/outputs/EtaAssignedOutputEvent.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  }
+
+  public void setIdentifier(String identifier) {
+    this.identifier = identifier;
+  }
+
+  private String identifier;
+
+  public EtaAssignedOutputEvent() {
+    type = Type.ETA;
```
Comment (ewbao) [inline cloud-functions/src/main/java/com/google/fleetevents/odrd/models/outputs/EtaAssignedOutputEvent.java:43]: Maybe add a new type?

File: beam/src/main/java/com/google/fleetevents/beam/TaskOutcomeChange.java [hunk 3]
```diff
@@ -1,17 +1,3 @@
-// Copyright 2023 Google Inc.
```
[the remark this answers is not in the document]
Comment (gmalmiski) [reply beam/src/main/java/com/google/fleetevents/beam/TaskOutcomeChange.java:1]: No wasn't merged previously added back.

File: cloud-functions/src/main/java/com/google/fleetevents/lmfs/models/outputs/EtaAssignedOutputEvent.java (hunk 1 above)
Comment (gmalmiski) [reply cloud-functions/src/main/java/com/google/fleetevents/lmfs/models/outputs/EtaAssignedOutputEvent.java:23]: Added new type ETA_ASSIGNED

**Review by ewbao — APPROVED** [review]

**Pull request merged**